packages/ingest/ftp.js

```diff
@@ -174,7 +175,12 @@ module.exports.ftpMixin = (superclass) => class extends superclass {
     const pass = new PassThrough();
     readable.pipe(pass);
 
-    const params = { Bucket: bucket, Key: key, Body: pass };
+    const params = {
+      Bucket: bucket,
+      Key: key,
+      Body: pass,
+      ContentType: mime.lookup(key) || null
```

Comment:
Since `ftp` doesn't really have a `mime-type`, try to infer. If we can't infer, just let aws set the default `ContentType` value.

Reply:
Re-reading this, I don't quite understand why the protocol matters for mime-type inference if `mime.lookup` is just inspecting the file extension of the S3 key?

Reply:
Protocol doesn't matter once we get this far. Http has one step earlier on that attempts to set the `ContentType` based on the information in the header.